test/fuzz/luaL_loadbuffer/luaL_loadbuffer_fuzzer.cc

```diff
@@ -34,7 +34,7 @@ DEFINE_PROTO_FUZZER(const lua_grammar::Block &message)
 	if (!L)
 		return;
 
-	std::string code = BlockToString(message);
+	std::string code = MainBlockToString(message);
 
```

Comment:
Please add to commit message that idea was derived from a paper "Program Reconditioning: Avoiding Undefined Behaviour When Finding and Reducing Compiler Bugs" and add a link to a paper https://www.doc.ic.ac.uk/~afd/homepages/papers/pdfs/2023/PLDI.pdf
Also, please describe a general idea there.